src/main/resources/edu/cornell/kfs/coa/document/workflow/SubObjectCodeGlobalEditMaintenanceDocument.xml

```diff
@@ -19,7 +19,7 @@
     <documentType>
       <name>GSOI</name>
       <parent>COA</parent>
-      <label>Global Sub-Object Code Inactivation</label>
+      <label>Sub-Object Code Global Inactivation</label>
```

Comment:
The code is able to automatically generate the link label from the label of the doc type. In this case the label was inconsistent, so the easiest way to get the correct link label was to change the doc type xml and let the code do it's thing (otherwise I would have had to add several custom lines to the json). We had previously made a similar change to another global doc for consistency as well.